distributor/cmd/main.go

```diff
@@ -236,6 +366,14 @@ func forwardEventToAPI(event cloudevents.Event, apiEndpoint string) error {
 	return errors.New(string(body))
 }
 
+func getHTTPClient() *http.Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: env.VerifySSL},
```

Comment:
The naming of the environment variable VerifySSL is misleading because I would set it to `false` to disable it, but it determines whether to skip SSL verification.

Reply:
We have set it to `HTTP_SSL_VERIFY` in dynatrace-service. Let's use the same naming here.

https://github.com/keptn-contrib/dynatrace-service/blob/81e8ff9dfef995e4cc42411e80e4353d59405fec/deploy/service.yaml#L126-L127

Reply:
I think this was resolved in https://github.com/keptn/keptn/pull/2884/commits/31084d362f077430c956ae9fd98b063a691e533f